tests/unit/tls_test.cc

```diff
@@ -157,44 +186,54 @@ SEASTAR_TEST_CASE(test_x509_client_with_priority_strings) {
         "SECURE256:+SECURE128",
         "NORMAL:%COMPAT",
         "NORMAL:-MD5",
-        "NONE:+VERS-TLS-ALL:+MAC-ALL:+RSA:+AES-128-CBC:+SIGN-ALL:+COMP-NULL",
+        "NONE:+VERS-TLS-ALL:+MAC-ALL:+RSA:+AES-256-GCM:+SIGN-ALL:+COMP-NULL:+GROUP-ALL",
```

Comment:
Seems unrelated to rest of change.

Reply:
it's related. i am quoting part of the commit message here

>     because the HTTPS server started by https-server.py uses ECDHE
>     for key exchange, we need to enable a elliptic curves group. please
>     note, according to
>     https://gnutls.org/manual/html_node/Priority-Strings.html,
>     NORMAL enables both ECDHE, DHE and plain RSA. and since ECDHE
>     is more secure than RSA, let's use ECDHE for testing.

Reply:
Fair enough, but adding "GROUP-ALL" just enabled all default things, and more or less makes the prio part before it pointless, no? Not really testing any specific combo at all...

Reply:
> Fair enough, but adding "GROUP-ALL" just enabled all default things,

i don't think so. "GROUP-ALL" is not "NORMAL". it only enables all groups for ec and finite field groups.

Reply:
Docs: `Catch all which enables all groups from NORMAL priority is GROUP-ALL`. I may be wrong, but it sounds awfully like "NORMAL" to me...
`GROUP-EC-ALL` enables all EC variants.

Reply:
> Docs: `Catch all which enables all groups from NORMAL priority is GROUP-ALL`. I may be wrong, but it sounds awfully like "NORMAL" to me...

please note, GROUP-ALL does not imply `KX-ALL` or `CIPHER-ALL`. again, it only enables the listed groups in the document.